tests/Unit/Domain/CoordinateMaps/Test_CoordinateMap.cpp

```diff
@@ -586,34 +619,73 @@ void test_make_vector_coordinate_map_base() {
               OrientationMap<2>{std::array<Direction<2>, 2>{
                   {Direction<2>::lower_eta(), Direction<2>::upper_xi()}}},
               true});
-  const auto upper_xi_wedge =
-      make_coordinate_map<Frame::Logical, Frame::Inertial>(Wedge2DMap{
-          1.0, 2.0, 0.0, 1.0,
-          OrientationMap<2>{std::array<Direction<2>, 2>{
-              {Direction<2>::upper_xi(), Direction<2>::upper_eta()}}},
-          true});
-  const auto upper_eta_wedge =
-      make_coordinate_map<Frame::Logical, Frame::Inertial>(Wedge2DMap{
-          1.0, 2.0, 0.0, 1.0,
-          OrientationMap<2>{std::array<Direction<2>, 2>{
-              {Direction<2>::upper_eta(), Direction<2>::lower_xi()}}},
-          true});
-  const auto lower_xi_wedge =
-      make_coordinate_map<Frame::Logical, Frame::Inertial>(Wedge2DMap{
-          1.0, 2.0, 0.0, 1.0,
-          OrientationMap<2>{std::array<Direction<2>, 2>{
-              {Direction<2>::lower_xi(), Direction<2>::lower_eta()}}},
-          true});
-  const auto lower_eta_wedge =
-      make_coordinate_map<Frame::Logical, Frame::Inertial>(Wedge2DMap{
-          1.0, 2.0, 0.0, 1.0,
-          OrientationMap<2>{std::array<Direction<2>, 2>{
-              {Direction<2>::lower_eta(), Direction<2>::upper_xi()}}},
-          true});
-  CHECK(upper_xi_wedge == *(vector_of_wedges[0]));
```

Comment:
These checks are replaced by lines 623-630, since I changed the type of upper_xi_wedge, etc from CoordinateMap to CoordinateMaps::Wedge2D

Reply:
but now you are comparing vector_of_wedges to CoordinateMapBases instead of CoordinateMaps?

Reply:
I can do the check for both, I just wasn't sure what would be count as sufficiently checking since I assumed CoordinateMapBases and CoordinateMaps only differed by a use of `*`